src/main/java/frc/robot/RobotContainer.java

```diff
@@ -66,7 +69,7 @@ private Command climb(boolean trap) {
         return new SequentialCommandGroup(m_elevator.climbFactory().alongWith(m_climber.prepArmsFactory())
                 .until(m_elevator::isElevatorUp),
                 m_climber.climbFactory().alongWith(trap ? m_elevator.trapFactory() : m_elevator.climbFactory()))
-                .unless(() -> drivetrain.tipping().getAsBoolean());
+                .unless(drivetrain.tipping());
```

Comment:
I think this should be:
`.unless(drivetrain::tipping)`

Reply:
my bad...this is fine. Triggers are boolean suppliers. Got it. That will eventually stick.